openoffice.txt

```diff
@@ -11,7 +11,7 @@ SHA1 = 68d73772b0f9559a08d65ca556230fe32530af41
 CDPath = none
 SizeBytes = 142587736
 Icon = openoffice.ico
-Screenshot1 = https://home.apache.org/~mseidel/AOO-ReactOS.png
+Screenshot1 = https://imgur.com/a/yPI7VWG
```

Comment:
```suggestion
Screenshot1 = https://i.imgur.com/ADdiCqE.png
```
You linked to a webpage containing an image, this is a direct image link.

Reply:
Already solved...